source/playgrounds.txt

```diff
@@ -30,12 +30,25 @@ files with the ``.mongodb`` extension as playgrounds.
 
 .. include:: /includes/admonitions/note-playgrounds.rst
 
-|vsce| splits your Playground window to display the results of your 
-Playground in |json| format in the right-side pane labeled ``Playground 
-Results.json``. If you disabled split-view, |vsce| displays the 
-Playground results in |json| format in a new tab to the right of 
-your Playground. You cannot edit the results in the results pane. 
-You can save the results to a file. 
+|vsce| splits your Playground window to display the results of your
+Playground in |json| format in the right-side pane labeled ``Playground
+Results.json``.
+
+If you disabled split-view, |vsce| displays the Playground results in
+|json| format in a new tab to the right of your Playground.
+
+In the Playground results tab, you can:
+
+- Save the results to a file.
+- Edit the results in the results pane if you made a
+  :doc:`findOne or find query </read-document-playground>` in the
+  Playground.
+
+  1. At the top of the results tab, click :guilabel:`Edit Document`.
```

Comment:
If you compare the `Edit Document` code lens with the `Currently connected to` code lens that we have in a playground, you will see that the `Currently connected to` code lens always appears at line 0 of the editor, and in this case, it would be correct to say that the code lens appears `at the top of the results tab`.
But the `Edit Document` code lens appears not `at the top of the results tab`, it appears `at the top of the json document` (if a playground returned a single document as a result), or `at the top of each json document` (if a playground returned a list of documents as a result).

Reply:
**question:** Based on your prior comments, where would the **Edit Document** link appear on a non-JSON document?

Reply:
The `Edit Document` link appears only on JSON documents.